appveyor.yml

```diff
@@ -0,0 +1,16 @@
+clone_folder: C:\projects\neon-lang\
```

Comment:
This shouldn't be necessary. If you can, remove this line and the `--directory=` option to scons (unless there is a good reason that this *needs* to be explicitly specified).